src/test/java/baza/trainee/controller/EventAdminControllerTest.java

```diff
@@ -0,0 +1,117 @@
+package baza.trainee.controller;
+
+import static org.mockito.Mockito.*;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.*;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.*;
+
+import baza.trainee.domain.dto.event.EventDto;
+import baza.trainee.domain.dto.event.EventPreviewDto;
+import baza.trainee.domain.dto.event.EventPublicationDto;
+import baza.trainee.service.EventService;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.List;
+
+@WebMvcTest(EventAdminController.class)
+public class EventAdminControllerTest {
```

Comment:
Did u try to run your tests?
I have next error 
"java: java.lang.IllegalStateException: Duplicate key getBytes (attempted merging values getBytes(java.lang.String) and getBytes(java.nio.charset.Charset))

Reply:
now i have new test, there are not many, but all work